FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_covg/multiple_priorities_no_timeslice_covg_utest.c

```diff
@@ -0,0 +1,630 @@
+/*
```

Comment:
Just a general comment for all the new tests in the smp/ folder.
I believe each *_covg_utest.c file can be moved into the matching folder for that build configuration. This would reduce the number of Makefiles/config.h files to maintain and provide a better logical grouping where each sub-folder under smp/ is a build configuration. If there is a build configuration we need, but its not found in smp/ it either means we missed something in our test plan or the coverage tests belong in another root folder.